python/datasources/data_source.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC
+from ingestion import di_url_file_to_gcs, gcs_to_bq_util
+
+
+# Abstract base class for all data sources ingested by the Health Equity Tracker.
+# This also includes default implementations for each of the ingestion methods.
+class DataSource(ABC):
```

Comment:
Should we add a @abstractmethod @staticmethod get_id()?

Reply:
Done

Reply:
Nice. I was thinking we could add the `@abstractmethod` decorator but not a strong opinion.